apps/help-center/help-center-gutenberg-disconnected.js

```diff
@@ -15,12 +22,23 @@ function HelpCenterContent() {
 				href="https://wordpress.com/help"
 				icon={ <HelpIcon /> }
 				label="Help"
-				size="compact"
+				size={ canvasMode === 'edit' ? 'compact' : undefined }
```

Comment:
I just noticed that it looks like the button is no longer sized to `compact` in the post editor's header bar at the top right:

<img width="362" alt="image" src="https://github.com/user-attachments/assets/e4338b60-6e35-4bd4-89c5-42b136ec43fa">

I think in that case it should still be `size="compact"`. Is there another way to skip the compact sizing when it's required so that the buttons in the post editor header can line up?

I've logged this in a separate issue over in #93976. I couldn't find an existing issue for it, so apologies if that's a double up!

Reply:
Oops...the canvas mode seems to be unavailable in the post editor. Will check it later 👍

Reply:
Yes, the canvas mode only exists conceptually with the site editor for now.

> Will check it later 👍

Feel free to add me as a reviewer if you open a PR for it!